<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window_Search</class>
 <widget class="QDialog" name="Window_Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>271</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_BookName">
        <property name="text">
         <string>BookName</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_BookName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_AuthorName">
        <property name="text">
         <string>Author</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_AuthorName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_Subject">
        <property name="text">
         <string>Subject</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Subject"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Book ID </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_BookId">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ID">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Search By ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_name">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Search By Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_author">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Search By Author</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_subject">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Search By Subject</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
